# Fluent value for task planner's Fluent

# constants for type
uint8 STRING = 0
uint8 INT = 1

uint8 data_type
string data
